dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -157,6 +157,7 @@ void main() {
               AndroidSemanticsAction.copy,
               AndroidSemanticsAction.setSelection,
               AndroidSemanticsAction.setText,
+              AndroidSemanticsAction.nextAtMovementGranularity,
```

Comment:
If enter text sets the selection to the end , why there is only nextAtMovementGranularity but no previousAtMovementGranularity?

Reply:
not sure this is what the error message told me to do.

Reply:
This is a bit confusing to me: https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#ACTION_NEXT_AT_MOVEMENT_GRANULARITY

The example says "Move to the previous character"

Reply:
We added semantics action in RenderEditable.describeSemanticsConfiguration. These action will propagate to AccessibilityBridge.java and the real android semantics action will be added in createAccessibilityNodeInfo in that file which will then show up in the test here. I think we should figure out why there is nextAtMovementGranularity because the RenderEditable.describeSemanticsConfiguration should only add previousAtMovementGranularity

Reply:
Ok there were 2 typos. Fixed now.